docs/modules/ROOT/pages/reference/cli.adoc

```diff
@@ -0,0 +1,98 @@
+= Command Line Options
+
+== Generic Options
+
+Some command line options aren't associated with a specific command, but can
+provided to any command instead.
+
+*--version*::
+  Show the version and exit.
+
+*-v, --verbose*::
+  Control verbosity. Can be repeated for more verbose output.
++
+Currently three verbosity levels are implemented, regular operation is
+active when the flag isn't provided, debug verbosity is enabled when the
+flag is provided once or more, and trace verbosity is enabled when the flag
+is provided three times or more.
+
+*--help*::
+  Show a help message and exit.
+
+== Catalog Clean
+
+This command doesn't have any command line options.
+
+== Catalog Compile
+
+*--api-url* URL::
+  xref:lieutenant:ROOT:index.adoc[Lieutenant] API URL.
+
+*--api-token* TOKEN::
+  Lieutenant API token.
+
+*--global-git-base* URL::
+  Base directory for global Git config repositories.
+
+*--local*::
+  Run in local mode, local mode doesn't try to connect to the Lieutenant API or
+fetch/push Git repositories.
+
+*--push*::
+  Push catalog to remote repository.
+
+*--git-author-name* USERNAME::
+  Name of catalog commit author
+
+*--git-author-email* EMAIL::
+  E-mail address of catalog commit author
+
+*-v, --verbose*::
+  Control verbosity. Can be repeated for more verbose output.
+
+*--help*::
+  Show a help message and exit.
+
+== Component Compile
+
+*-f, --values* FILE::
+  Specify inventory class in a YAML file. This option can be repeated to
+  provide multiple files. Files specified later win when resolving inventory
+  values.
+
+*-J, --search-paths* DIRECTORY::
+  Specify additional search paths.
+
+*-o, --output* DIRECTORY::
+  Specify output path for compiled component. Defaults to `./`.
+
+*-v, --verbose*::
+  Control verbosity. Can be repeated for more verbose output.
+
+*--help*::
+  Show help message and exit.
+
+== Component New
+
+*--name* TEXT::
+  The component's name as it will be written in the documentation. Defaults to the slug.
+
+*--lib / --no-lib*::
+  Add a component library template. Defaults to _no_.
+
+*--pp / --no-pp*::
+  Add a component postprocessing template. Defaults to _no_.
+
+*--owner* TEXT::
+The GitHub user or project name where the component will be hosted. Defaults
+to _projectsyn_.
+
+*--copyright* TEXT::
+The copyright holder added to the license file. Defaults to "VSHN AG <info@vshn.ch>".
+
+*-v, --verbose*::
+  Control verbosity. Can be repeated for more verbose output.
+
+*--help*::
```

Comment:
Isn't that already mentioned in "Generic Options"?

Reply:
yep, I've removed the redundant entries in the latest version, and expanded the documentation under generic options to mention that `-v` can be given in different positions.